ui/app/services/auth.js

```diff
@@ -417,32 +473,6 @@ export default Service.extend({
     this.set('tokens', tokenNames);
   },
 
-  // returns the key for the token to use
-  currentTokenName: computed('activeCluster', 'tokens', 'tokens.[]', function () {
-    const regex = new RegExp(this.activeCluster);
-    return this.tokens.find((key) => regex.test(key));
-  }),
-
-  currentToken: computed('currentTokenName', function () {
-    const name = this.currentTokenName;
-    const data = name && this.getTokenData(name);
-    // data.token is undefined so that's why it returns current token undefined
-    return name && data ? data.token : null;
-  }),
-
-  authData: computed('currentTokenName', function () {
-    const token = this.currentTokenName;
-    if (!token) {
-      return;
-    }
-    const backend = this.backendFromTokenName(token);
-    const stored = this.getTokenData(token);
-
-    return assign(stored, {
-      backend: BACKENDS.findBy('type', backend),
-    });
-  }),
-
```

Comment:
These deletions are just from some reorganization to group the computed's together